custom/enikshay/nikshay_datamigration/management/commands/delete_migrated_cases.py

```diff
@@ -0,0 +1,75 @@
+from itertools import chain
+
+from django.core.management import BaseCommand
+
+from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
+from custom.enikshay.case_utils import (
+    CASE_TYPE_DRTB_HIV_REFERRAL,
+    CASE_TYPE_EPISODE,
+    CASE_TYPE_OCCURRENCE,
+    CASE_TYPE_PERSON,
+)
+
+
+def _get_parent_case_by_type(domain, child_case_id, parent_case_type):
```

Comment:
It looked like those methods only select open cases - I wanted to delete closed cases too.